web/backend/src/controllers/dela.ts

```diff
@@ -265,6 +265,5 @@ delaRouter.use('/*', (req, res) => {
   }
 
   const dataStr = JSON.stringify(bodyData);
-
```

Comment:
is this change on purpose? if not please remove it from the PR :)

Reply:
Indeed I will revert it 👍